src/main/java/uk/gov/companieshouse/certificates/orders/api/validator/RequestValidator.java

```diff
@@ -39,14 +40,28 @@
             return errors;
         }
         errors.addAll(getCollectionDeliveryValidationErrors(options));
-        if (options.getCertificateType() == DISSOLUTION_LIQUIDATION) {
-            if (options.getIncludeCompanyObjectsInformation()) {
+
+        if (options.getCertificateType() == DISSOLUTION) {
+
+            if (options.getIncludeCompanyObjectsInformation() != null && options.getIncludeCompanyObjectsInformation()) {
+                errors.add(
+                    "include_company_objects_information: must not be true when certificate type is dissolution");
+            }
+            if (options.getIncludeGoodStandingInformation() != null && options.getIncludeGoodStandingInformation()) {
```

Comment:
Does this still work if you remove && `options.getIncludeGoodStandingInformation()`

Reply:
It does, now removed